src/main/java/com/google/sps/api/CalendarClientHelper.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.sps.api;
+
+import com.google.api.services.calendar.model.Event;
+import com.google.api.services.calendar.model.EventAttendee;
+import java.io.Serializable;
+import java.util.List;
+
+public class CalendarClientHelper implements Serializable {
+  public CalendarClientHelper() {
+  }
+
+
+
+  /**
+   * This is a method used in CalendarInterface to filter for events that are attended by the user
+   * The EventAttendee's getSelf() method indicates whether this entry represents the calendar on which this copy of the event appears.
+   */
+  public static boolean isAttending(Event event) {
+    List<EventAttendee> attendeeList = event.getAttendees();
+    if (attendeeList != null) {
```

Comment:
In case it is possible, it is worth to use early-return. So:
```
if (attendeeList == null) {
  return true;
}
for ([...])
```
This way the reader can clearly see edge-cases that are handled at the beginning. They don't need to scroll back and forth to see "which `if` this `else` clause match?" in case of long body. Also indentation level is reduced.